### DISCUSSION TITLE
Holberton Lesotho Git Discussions

### DISCUSSION THREAD

**Discussion by ragmer09toske** [community/community#106465] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

ask me anything here so i could be able to help you anytime

**Comment by Fuma1322** [comment]
I hear you

**Comment by KamoheloKoali** [comment]
ask it hear

**Comment by Matross-20** [comment]
am hear too

**Comment by khiba-k** [comment]
i too am also here

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @ragmer09toske  Happy to have you here 🙂

You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed and locked to keep our community nice and tidy 🧹.
